src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private static final int FIGURE_COUNT = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private int number;
+    private String color;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public void setColor(ColorSupplier color) {

```

Comment:
Try to not leave methods that aren't used. As you change your logic and use constructor you don't need to use set methods anymore